docs/reference/sql/endpoints/rest.asciidoc

```diff
@@ -270,8 +273,8 @@ cursor: "sDXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAAEWWWdrRlVfSS1TbDYtcW9lc1FJNmlYdw==:BAFmB
 [[sql-pagination]]
 === Paginating through a large response
 
-Using the example above, one can continue to the next page by sending back the `cursor` field. In
-case of text format the cursor is returned as `Cursor` http header.
+Using the example from the <<sql-rest-format,previous section>>, one can
+continue to the next page by sending back the cursor field.
```

Comment:
Please, also, keep the second sentence as well. The final text should be

> Using the example from the <<sql-rest-format,previous section>>, one can continue to the next page by sending back the `cursor` field. In case of text format the cursor is returned as `Cursor` http header.

Reply:
Thanks for catching!